lib/icingadb/icingadb-utility.cpp

```diff
@@ -62,7 +62,13 @@ String IcingaDB::FormatCommandLine(const Value& commandLine)
 
 String IcingaDB::GetObjectIdentifier(const ConfigObject::Ptr& object)
 {
-	return HashValue(new Array({m_EnvironmentId, object->GetName()}));
+	String identifier = object->GetIdentifier();
+	if (identifier.IsEmpty()) {
+		identifier = HashValue(new Array({m_EnvironmentId, object->GetName()}));
+		object->SetIdentifier(identifier, true);
```

Comment:
Errm... what's _true_ again?

Reply:
Suppresses any kind of events for this attribute, but in this case the `OnIdentifierChanged` event.

Reply:
Do we need this?

Reply:
Could spare us some CPU time :)

Reply:
For sure there are reasons for _true_. That isn't any of them IMAO.